draft-packages/pagination/KaizenDraft/styles.scss

```diff
@@ -0,0 +1,125 @@
+@import "~@kaizen/design-tokens/sass/typography-vars";
+@import "~@kaizen/design-tokens/sass/color-vars";
+@import "~@kaizen/design-tokens/sass/border-vars";
+@import "~@kaizen/component-library/styles/border";
+@import "~@kaizen/component-library/styles/type";
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.arrowIconWrapper {
+  height: 36px;
+  width: 36px;
+  border-radius: 18px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: $kz-var-border-focus-ring-border-width solid $kz-var-color-cluny-200;
```

Comment:
Your decision to make, but `$kz-var-border-focus-ring-border-style` can be used in place of `solid` (but maybe you want `solid` to be declared explicitly).
https://cultureamp.design/storybook/?path=/story/design-tokens-api--page